pkg/kubectl/run.go

```diff
@@ -368,6 +368,9 @@ func (CronJobV2Alpha1) Generate(genericParams map[string]interface{}) (runtime.O
 			Schedule:          params["schedule"],
 			ConcurrencyPolicy: batch.AllowConcurrent,
 			JobTemplate: batch.JobTemplateSpec{
+				ObjectMeta: api.ObjectMeta{
+					Labels: labels,
```

Comment:
Why do we need this change? Note that by default cronjob controller doesn't set `jobTemplate.labels`, and it will be copied from `jobTemplate.spec.template.labels` upon job creation